### PR TITLE
chore: prepare for 1.0.0

### PR DESCRIPTION
**Pull request by lstocchi**
@mohitsuman I just updated the README following what was written before. As it didn't mention anything about actions i didn't add any new section. Should I create one and write something about the actions we added (deploy, build, run ...)?
Sudhir is on PTO so I guess you're the only one who can review it

### CODE REVIEW

**Comment by mohitsuman** [comment]
@lstocchi yes we should update the readme with the actions info. I will review it accordingly.

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "vscode-knative",
   "displayName": "Knative",
   "description": "Knative extension for VSCode",
-  "version": "0.9.4",
+  "version": "1.0.0",
   "license": "MIT",
   "publisher": "redhat",
   "preview": true,
```
Comment (sudhirverma) [inline package.json:8]: should we remove the preview tag?

Comment (lstocchi) [reply package.json:8]: @mohitsuman

Comment (mohitsuman) [reply package.json:8]: For serverless, let us do a few more releases and then move out of the `Preview` tag.

**Review by mohitsuman — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ All notable changes to the "vscode-knative" extension will be documented in this
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.0 (2022/02/21)
+
+- Add support for Knative Functions
+  - Add dependency to func cli
+  - New Functions view
+  - Add actions to work with them (build, deploy, run, undeploy, open in browser, configure)
```
Comment (sudhirverma) [inline CHANGELOG.md:13]: create function is missing.

**Pull request merged**